test/testlib/nuodb_database_utilities.go

```diff
@@ -625,14 +625,18 @@ func PurgeDatabaseArchives(t *testing.T, namespaceName string, adminPod string,
 	}
 }
 
-func SuspendDatabaseBackupJobs(t *testing.T, namespaceName string, dbName string, backupGroup string) {
+func SuspendDatabaseBackupJobs(t *testing.T, namespaceName string, domain, dbName string, backupGroup string) {
 	kubectlOptions := k8s.NewKubectlOptions("", "", namespaceName)
-	cronJobs := ListCronJobs(t, namespaceName)
-	expectedSuffix := fmt.Sprintf("-hotcopy-%s-%s", dbName, backupGroup)
+	// the CronJob apiVersion may be different depending on the Kubernetes version
+	output, err := k8s.RunKubectlAndGetOutputE(t, kubectlOptions, "get", "cronjobs", "-oname")
+	require.NoError(t, err)
+	cronJobs := strings.Split(strings.ReplaceAll(output, "\r\n", "\n"), "\n")
```

Comment:
Is it insufficient to just split by any whitespace?

Reply:
This is splitting by new line `\n` only. It just replaces the Windows variant before that.